src/implementations/twig/components/notification/notification.story.js

```diff
@@ -67,6 +84,9 @@ const prepareData = (data, args) => {
   const clone = JSON.parse(JSON.stringify(data));
 
   // Optional elements
+  if (!args.show_title) {
+    delete clone.title;
```

Comment:
it seems this control is not working, doesn't hide the title, could it be a conflict that i solved not in the right way..?